x-team/xp-ui
# Add disabled state on ConfirmationBox

**Release Type:** *Non-breaking changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-3750

## Description

Add disabled state on ConfirmationBox

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3749-delete-experience` | [XP API](https://github.com/x-team/xp-api/pull/829)
`XP-3750-implemente-work-experience-delete-functionality` | [XP Registration](https://github.com/x-team/xp-registration/pull/203)

## Steps to Test or Reproduce

1. Go to http://localhost:9001/?path=/story/core-components-confirmationbox-states--disabled
1. Observe that the buttons are disabled

<img width="477" alt="Screen Shot 2020-05-25 at 20 28 39" src="https://user-images.githubusercontent.com/131859/82847773-5aa99d80-9ec6-11ea-8a38-a8f689441d75.png">
